src/css/components/Modal.scss

```diff
@@ -22,18 +22,28 @@
   min-width: 50%;
   height: auto;
   max-height: 90vh;
-  overflow: scroll;
+  overflow: hidden;
```

Comment:
This will force the modal-body to scroll instead of the main modal container, allowing the header to be fixed and the scroll bar to be inset, preserving the rounded corner appearance.